code/game/machinery/doors/firedoor.dm

```diff
@@ -267,9 +270,6 @@ var/global/list/alert_overlays_global = list()
 		if(check_access(ID))
 			access_granted = 1
 
-	var/answer = "Yes"
-	if(answer == "No")
-		return
```

Comment:
Probably put it in the wrong file and left it unfinished, it reminds me of the AI warning when it attempts to force open an airlock.